src/modules/web3/model/funds-deposited-ev.entity.ts

```diff
@@ -0,0 +1,48 @@
+import { Column, CreateDateColumn, Entity, PrimaryGeneratedColumn, Unique, UpdateDateColumn } from "typeorm";
+
+export type FundsDepositedEventArgs = {
+  amount: string;
+  originChainId: string;
+  destinationChainId: string;
+  relayerFeePct: string;
+  depositId: number;
+  quoteTimestamp: number;
+  originToken: string;
+  recipient: string;
+  depositor: string;
+};
+@Entity({
+  schema: "events",
+})
+@Unique("UK_funds_deposited_ev_transactionHash_logIndex", ["transactionHash", "logIndex"])
+export class FundsDepositedEv {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  blockNumber: number;
+
+  @Column()
+  blockHash: string;
+
+  @Column()
+  transactionIndex: number;
+
+  @Column()
+  address: string;
+
+  @Column()
+  transactionHash: string;
+
+  @Column()
+  logIndex: number;
+
+  @Column({ type: "jsonb" })
+  args: FundsDepositedEventArgs;
+
+  @CreateDateColumn()
+  createdAt: Date;
+
+  @UpdateDateColumn()
+  updatedAt: Date;
```

Comment:
I think adding the field `chainId` would make sense, no?

Reply:
yes. Adding it now

Reply:
Added [here](https://github.com/across-protocol/scraper-api/pull/224/commits/8bdf4c08e18be0d1fd039e1807e60e6c06465062)